packages/campfhir-cli/src/commands/assistant.ts

```diff
@@ -22,35 +28,60 @@ export default <CommandModule>{
   handler: async (_options) => {
     const examples = await loadTrainingData();
     const exampleFormatterTemplate = "Q: {prompt} ||| api: {completion}";
+    const instructions = `** INSTRUCTIONS **
+The only known api classes valid for answer are: Patient, RiskAssessment, Practitioner, Appointment, CarePlan.
+All other classes are unknown.
+Classes names are case insensitive. There are no subclasses or roles derived from the known classes.
+If you are asked for an unknown class you should answer: "Sorry, I don't know about CLASS", interpolating "CLASS" with the unknown class name.
```

Comment:
This is both fantastic and deeply weird.
I am amazed though by the facilities offered by langchain.

Reply:
I second _weird_, it sometimes has more in common with _reverse psychology_ than _engineering_